model/handlers.py

```diff
@@ -98,6 +98,16 @@
     diff = await respond(mutated_model, message, mutated_model_id, wild_type_model_id=wild_type_id)
     return web.json_response(diff)
 
+
+async def model_info(request):
+    wild_type_id = request.match_info['model_id']
+    wild_type_model = Models.get(wild_type_id)
+    return web.json_response({'medium': [dict(
```

Comment:
For me this looks like a bit too concise. How about breaking it down into two steps: assembling the data + doing the final transformation?

    medium = []
    for reaction_id in wild_type_model.medium:
        medium.append({
            'id': reaction_id,
            'name': wild_type_model.reactions.get_by_id(reaction_id).name.replace('exchange', '').strip(),
        })
    return web.json_response({'medium': medium})

Is it necessary to wrap this response in `'medium'`? This is not really restful, and on the client side it causes awkward situations like: `mediumResponse.medium` or `mediumData.medium` whereas with a flat structure it could be just `medium`.

Reply:
I wrapped the response to medium for it to have the same format as MapInfo we use for Data Driven cards https://github.com/DD-DeCaF/metabolica-ui-map/blob/master/src/pathwayvis/types.ts#L49 so it can be used in the same manner

Reply:
ok, then it's better like this, but I think we should change both of them over time.